custom_components/better_thermostat/climate.py

```diff
@@ -257,36 +257,46 @@ def _async_startup(*_):
 			self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, _async_startup)
 		
 		# Check If we have an old state
-		old_state = await self.async_get_last_state()
-		if old_state is not None:
+		if (old_state := await self.async_get_last_state()) is not None:
 			# If we have no initial temperature, restore
 			if self._target_temp is None:
 				# If we have a previously saved temperature
 				if old_state.attributes.get(ATTR_TEMPERATURE) is None:
-					self._target_temp = self.min_temp
+					self._target_temp = self._TRV_min_temp
```

Comment:
Nope. We want to set our minimum temperature, not the minimum temperature of the TRV.